src/rard/templates/research/partials/render_inline_worklink_form.html

```diff
@@ -0,0 +1,23 @@
+{% load i18n bootstrap4 %}
+
+<div class="inline-worklink__form" hx-target="this" hx-swap="outerHTML">
+    <form novalidate enctype="form-data" autocomplete='off' action='{{ request.path }}' hx-post='{% url inline_update_url link.pk %}'>
+    {% csrf_token %}
+    <div class="inline-worklink__form-field">
+       <p> {{form.instance.antiquarian}}</p>
+        {% bootstrap_field form.definite_antiquarian field_class="toggle-definite" %}
+    </div>
+    <div class="inline-worklink__form-field">
+        <p>{{form.instance.work.name}}</p>
+        {% bootstrap_field form.definite_work field_class="toggle-definite" %}
```

Comment:
This form is letting me update antiquarian definite status and change book, but definite work and definite book don't stick.